### PR TITLE
Update to use Xcode 12 for CI and address any Xcode 12 issues

### PR DESCRIPTION
**Pull request by kaisong1990**

### CODE REVIEW

**Comment by oldalton** [comment]
@kaisong1990, @ameyapat - we shouldn't merge directly to master. Please create a "release/xxx" branch and merge into that one.

File: ADAL/ADAL.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -4854,7 +4854,7 @@
 				DEVELOPMENT_TEAM = UBF8T346G9;
 				ENABLE_BITCODE = NO;
 				GCC_OPTIMIZATION_LEVEL = 0;
-				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```
Comment (oldalton) [inline ADAL/ADAL.xcodeproj/project.pbxproj:4857]: please also update CocoaPods file

**Comment by ameyapat** [comment]
@oldalton @kaisong1990 
I didn't realize a PR was open on this branch, so when I committed to this branch it must have triggered a rebuild. I have further changes along with updates to deployment target that need to be committed. I'll close this PR and create a new branch "ameyapat/ADAL-ios-14" that will have further changes. A release branch "release/ADAL-ios-14" will also be created in which final changes will be merged from my branch.

**Comment by kaisong1990** [comment]
Feel free to delete this branch when all necessary changes are transferred to yours

**Pull request closed without merging**